### DISCUSSION TITLE
Concatenate parameters in test scenarios

### DISCUSSION THREAD

**Discussion by bbdsphe** [allure-framework/allure2#2496] [Ideas & Feedback]
**Is your feature request related to a problem? Please describe.**
I have an issue when I have to display a lot of parameters for a test scenario. The scenario name/description ends up being squashed to the left of the screen.

**Describe the solution you'd like**
Will it be possible to either add an option to remove parameters from the test scenario list (only allowing the parameters to be viewed on the right window, after clicking on the test scenario), or concatenate the parameters, prioritizing the name of the test scenario?

**Describe alternatives you've considered**
 \-

**Additional context**
\-

**Comment by noomorph** [comment]
This has to be moved to **Discussions**.

@bbdsphe, anyways, maybe you could try to disable that via a CSS override?

**Reply by bbdsphe** [reply]
That's a solid work around.

I'll give that go. Thanks for the tip!
